### PR TITLE
feat(vanilla-extract): migrate Callouts, Cards, Checkbox, Chip, Circle, CreditCard, CVV and DateTimePicker

### PR DESCRIPTION
**Pull request by sergivillar**

### CODE REVIEW

File: src/callout.tsx [hunk 1]
```diff
@@ -4,41 +4,18 @@ import Inline from './inline';
...
-    container: {
-        background: colors.backgroundAlternative,
-        borderRadius: 8,
-        padding: 16,
-        overflow: 'hidden',
-        display: 'flex',
-        minHeight: 56,
-        '& > *': {
-            flexShrink: 0,
-        },
```
Comment (atabel) [inline src/callout.tsx:27]: not needed?

File: src/card.css.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        },
+    },
+]);
+
+export const snapCard = style([
+    sprinkles({
+        display: 'flex',
+        height: '100%',
+
+        flexDirection: 'column',
```
Comment (atabel) [inline src/card.css.ts:66]:
```suggestion
        height: '100%',
        flexDirection: 'column',
```

File: src/card.tsx [hunk 3]
```diff
@@ -63,40 +56,52 @@ const CardContent: React.FC<CardContentProps> = ({
...
-                                        {pretitle}
-                                    </Text1>
-                                )}
-                                <Text4 truncate={titleLinesMax} as="h1" regular>
-                                    {title}
-                                </Text4>
-                                <Text2 truncate={subtitleLinesMax} as="div" regular>
-                                    {subtitle}
-                                </Text2>
+        <Stack space="between" className={sprinkles({display: 'flex', flex: 1})}>
```
Comment (atabel) [inline src/card.tsx:59]: hmmm... at this point, better use a div

File: src/card.css.ts [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+        '@media': {
+            [mq.desktopOrBigger]: {
+                padding: 24,
+            },
+        },
+    },
+]);
+
+export const snapCardTouchableBase = sprinkles({display: 'flex', height: '100%'});
```
Comment (atabel) [inline src/card.css.ts:83]:
```suggestion
const snapCardTouchableBase = sprinkles({display: 'flex', height: '100%'});
```

File: src/checkbox.css.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        flexShrink: 0,
+        borderRadius: 2,
+    }),
+    {
+        height: 18,
+        width: 18,
+        userSelect: 'none',
+        outline: '1px solid transparent',
+        verticalAlign: 'middle',
+        background: vars.colors.background,
```
Comment (atabel) [inline src/checkbox.css.ts:28]: you can use backgroundColor in sprinkles

File: src/chip.tsx [hunk 6]
```diff
@@ -77,14 +37,13 @@ type ChipProps = SimpleChipProps | ClosableChipProps | ToggleChipProps;
...
     const {texts, isDarkMode, colors} = useTheme();
-    const classes = useStyles({isDarkMode});
 
     const {Icon, children, id} = props;
 
     const body = (
         <>
             {Icon && (
-                <Box paddingRight={4} className={classes.icon}>
+                <Box paddingRight={4} className={classnames(props.active ? styles.iconActive : styles.icon)}>
```
Comment (atabel) [inline src/chip.tsx:46]:
```suggestion
                <Box paddingRight={4} className={props.active ? styles.iconActive : styles.icon}>
```

File: src/credit-card-number-field.tsx [hunk 7]
```diff
@@ -73,6 +73,7 @@ const CreditCardInput: React.FC<Props> = ({inputRef, value, defaultValue, onChan
 };
 
 const getAnimationTarget = (value?: string) => {
+    console.log({value});
```
Comment (atabel) [inline src/credit-card-number-field.tsx:76]: leftovers

File: src/callout.tsx (hunk 1 above)
Comment (sergivillar) [reply src/callout.tsx:27]: dont think so, only two direct children, `.content`, that already has a explicit `flex: 1`, and the icon, that has a fixed size... so I dont know what this think were doing... I tried it in storybook and all seems ok

File: src/checkbox.css.ts [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+export const checkboxContainer = style([
+    sprinkles({
+        display: 'inline',
+    }),
+    {
+        cursor: 'default',
+    },
+]);
+
+export const boxBase = style([
```
Comment (sergivillar) [inline src/checkbox.css.ts:14]: remove export!

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Pull request merged**